src/pages/basic_data/app_admin/InstitutionList.tsx

```diff
@@ -10,60 +14,112 @@ import {
   TableRow,
   Typography,
 } from '@mui/material';
+import { Box } from '@mui/system';
 import { visuallyHidden } from '@mui/utils';
+import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Link as RouterLink } from 'react-router-dom';
 import { alternatingTableRowColor } from '../../../themes/mainTheme';
 import { SimpleCustomerInstitution } from '../../../types/customerInstitution.types';
 import { dataTestId } from '../../../utils/dataTestIds';
-import { toDateString } from '../../../utils/date-helpers';
 import { getAdminInstitutionPath } from '../../../utils/urlPaths';
+import { SearchTextField } from '../../search/SearchTextField';
 
 interface InstitutionListProps {
   institutions: SimpleCustomerInstitution[];
 }
 
+enum InstitutionStatusFilter {
+  ShowAll,
+  Active,
+  Inactive,
+}
+
 export const InstitutionList = ({ institutions }: InstitutionListProps) => {
   const { t } = useTranslation();
 
+  const [customerStatus, setCustomerStatus] = useState<InstitutionStatusFilter>(InstitutionStatusFilter.ShowAll);
```

Comment:
gjør ingen skade, men typen sette automatisk basert på verdien du oppgir også
```suggestion
  const [customerStatus, setCustomerStatus] = useState(InstitutionStatusFilter.ShowAll);
```